File: Codeforces/C++/problemset/George and Accommodation.cpp
// Problem Link: https://codeforces.com/problemset/problem/467/A

#include <iostream>
using namespace std;
typedef long long ll;
int main()
{
    ll n, p, q, r = 0;
    cin >> n;
    for(ll i = 0; i < n; i++)
    {
        cin >> p >> q;
        if(p < q && (q-p) >= 2) r++;
    }
    cout << r;
    return 0;
}

File: src/main/kotlin/GeorgeandAccommodation.kt
package problems

/* Problem Link: https://codeforces.com/problemset/problem/467/A */

fun main() {
    val n = readln().toInt()
    var rooms = 0

    for (i in 1..n) {
        val (p, q) = readln().split(" ").map { it.toInt() }
        if (q >= p + 2) rooms++
    }

    print(rooms)
}